### DISCUSSION TITLE
AA Queries

### DISCUSSION THREAD

**Discussion by KrutikaMoorjani** [ONDC-Official/ONDC-FIS-Specifications#17] [Q&A] [answered]
**Sole Prop Flow**

1. As per Figma, AA journey will be shown within the buyer app. Lender just needs to create consent request on AA & pass the same to the buyer app through APIs. Is this understanding correct?

2. Will there by time lag for multiple lenders responding with the consents on AA?

3. Will the borrowers be shown multiple consent requests (for different lenders) on the same screen in the AA journey?

**Comment by nitins11** [comment]
1. Yes
3. It will be multiple consent requests in the beginning. Being worked out with AA to have one consent possible.

**Reply by KrutikaMoorjani** [reply, marked as answer]
Thanks Nitin
